src/ng/compile.js

```diff
@@ -3522,7 +3522,7 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
               }
               recordChanges(scopeName, newValue, oldValue);
               destination[scopeName] = newValue;
-            }, deepWatch);
```

Comment:
Deep watching never happens, but we still have a variable named  `deepWatch`. Isn't it a bit misleading?